tardis/tests/integration_tests/conftest.py

```diff
@@ -5,7 +5,7 @@
 import pytest
 
 from tardis import __githash__ as tardis_githash
-from tardis.tests.integration_tests.report import DokuReport
+#from tardis.tests.integration_tests.report import DokuReport
 from tardis.tests.integration_tests.plot_helpers import (
```

Comment:
@epassaro removing dokureport dependency will solve the issue of `self.contained` error. But now the data seems to be wrong as it gives error related to v_boundary being large.